app/src/main/res/xml/changelog_master.xml

```diff
@@ -3,6 +3,7 @@
     tools:ignore="MissingDefaultResource">
     <release version="2025.5.1 - Main" versioncode="3">
         <change>Add more Health Connect sensors</change>
+        <change>New design of To-do list widget</change>
         <change>Bug fixes and dependency updates</change>
```

Comment:
Do we include webSocket stability and crash fixed in this change?

Reply:
We generally don't write about specific fixes